src/bap_mc/bap_mc.ml

```diff
@@ -140,6 +158,19 @@ module Cmdline = struct
     let doc = "Output instruction kinds." in
     Arg.(value & flag & info ["show-kinds"] ~doc)
 
+  let show_oplen =
+    let formats = [
+	"hex", `hex;
+	"dec", `dec;
+      ] in
+    let doc = sprintf
+		"Output recognized opcode length, optional \
```

Comment:
Ok, yes show-size is more precise. Yeah we don't have to add hex or decimal, I will just remove those. Int of string will deserialize each case correctly anyway, so it is irrelevant. I will fix it.